### PR TITLE
Rename genAlignLoops

### PR DESCRIPTION
**Pull request by franksinankaya**
@am11 @jkotas @janvorli @sandreenko

### CODE REVIEW

**Comment by franksinankaya** [comment]
rerun coreclr ci?

File: src/jit/codegeninterface.h [hunk 1]
```diff
@@ -75,12 +75,12 @@ class CodeGenInterface
     //-------------------------------------------------------------------------
     //  The following property indicates whether to align loops.
     //  (Used to avoid effects of loop alignment when diagnosing perf issues.)
-    __declspec(property(get = doAlignLoops, put = setAlignLoops)) bool genAlignLoops;
-    bool doAlignLoops()
+
+    bool GetAlignLoops()
```
Comment (sandreenko) [inline src/jit/codegeninterface.h]: Please return it back to `DoAlignLoops'.

Comment (franksinankaya) [reply src/jit/codegeninterface.h]: still need a better name IMO, doAlignLoops() sounds like doing something whereas you are seeking for a bool value here. Something with IsXYZ?

Comment (jakobbotsch) [reply src/jit/codegeninterface.h]: `ShouldAlignLoops`?

Comment (franksinankaya) [reply src/jit/codegeninterface.h]: WFM. Consensus?

Comment (sandreenko) [reply src/jit/codegeninterface.h]: `ShouldAlignLoops` sounds better than `GetAlignLoops` or  `DoAlignLoops`, thanks @jakobbotsch.

**Pull request merged**